scripts/start

```diff
@@ -52,7 +52,7 @@ echo "Setting environment variables..."
 echo
 
 # Whitelist device IP, hostname and hidden service for CORS
-DEVICE_IP="$(hostname -I | cut -d ' ' -f 1)"
+DEVICE_IP="$(ip -o route get to 8.8.8.8 | sed -n 's/.*src \([0-9.]\+\).*/\1/p')"
```

Comment:
Will this still return the local IP on a local network that doesn't have internet access?

Reply:
Also, if we're using `ip` now we should probably add it to `check_dependencies`:

https://github.com/getumbrel/umbrel/pull/297/files#diff-ecb3d5615afcbe1e5df55695252c8f0835e8cb1ec10a6048a83cf0a80d2a1ab3R25

Reply:
Thanks, I added `ip` to the dependency check.

The method does not require internet access. The ip that is given in the command is never contacted, i.e. no network request is made. It just asks the system over which interface the request WOULD be routed if it is made, which should give you the IP address of your primary interface.

Reply:
Awesome TIL!